src/pages/homepage/homepage.js

```diff
@@ -1,18 +1,34 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import './homepage.css';
 import '../../components.css';
-import { Link } from 'react-router-dom';
+import { Link, useNavigate } from 'react-router-dom';
+import { useFilter } from '../../context/filterContext';
+import axios from 'axios';
 
 export default function Homepage() {
+
+    const [categories, setCategories] = useState();
+
+    useEffect(async () => {
+        const res = await axios.get("/api/categories");
+        console.log(res);
```

Comment:
console logs should be removed before pushing your code

Reply:
done